addons/payment_aps/__manifest__.py

```diff
@@ -4,8 +4,8 @@
     'name': "Payment Provider: Amazon Payment Services",
```

Comment:
That is a bit disturbing not to be able to read the provider name from the kanban view in the App Store. Can't we shorten the name and write it in the summary? (e.g. `Amazon Payment Services is an online [...]`) cc @AntoineVDV 
![image](https://user-images.githubusercontent.com/6265134/193775985-f5e49bc5-17ae-4faf-9ce0-d2a1d74d6b02.png)


```suggestion
    'name': "Payment Provider: APS",
```

Reply:
ok